.eslintignore

```diff
@@ -1,5 +1,4 @@
 dist
 node_modules
-schema
 **/*.tmpl.*
```

Comment:
I think this was meant to ignore `packages/schema/schema` but of course we do want to include `packages/schema` - maybe refine it instead?

```suggestion
schema/schema
**/*.tmpl.*
```

Reply:
Good point! I'll add `packages/schema/schema` for explicitness to the `.eslintignore` (even though it only contains .d.ts, .md and .json, which are not linted, but surely it's better not to rely on file extensions).